vllm/model_executor/layers/rotary_embedding.py

```diff
@@ -593,8 +586,6 @@ def forward(
                               torch.full_like(positions, k)).long()
         idx = (torch.add(positions, long_prompt_offset)
                if long_prompt_offset is not None else positions)
-        self.long_short_cos_sin_cache: torch.Tensor = (
-            self.long_short_cos_sin_cache.to(idx.device))
```

Comment:
restore these two lines and run again, we can see a nice error message telling us `long_short_cos_sin_cache` is the root cause.